.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This is a developer-specific file. Please don't commit this file (add it to .gitignore).

Reply:
Removed the file from control version